.github/workflows/go.yml

```diff
@@ -31,6 +31,18 @@ jobs:
           name: katzen.alpine
           path: ./katzen
 
+      - name: Build for Nix
+        run: make docker-build-nix
+
+      - name: Save output artifact name
+        run: ls nix_build | head -1 > nixos.output
```

Comment:
Is the package name sufficient to compare a locally built copy?

Reply:
That is, does Nix verify that the hash? in the package name matches the contents of the package?

Reply:
i would assume so? @ehmry can you confirm? and is there some standard way we should be publishing the output hash for each commit so that nix users can confirm their local build produced the same output as CI did?

Reply:
The package name is not an output hash, it is a hash on the input to the package, so yes, you should be hashing and publishing the output independently. As far as I know there isn't a standard way to publish output hashes. Nix users are just going to assume that the build is "probably deterministic" and not worry about it.